algorithms/kernel/stump/stump_train_impl.i

```diff
@@ -473,6 +473,7 @@ services::Status StumpTrainKernel<method, algorithmFPtype, cpu>::doStumpRegressi
     daal::tls<TGroupRes *> tls( [ = ]()-> TGroupRes *
     {
         TGroupRes *g = new TGroupRes();
+        DAAL_CHECK_STATUS_VAR(g)
```

Comment:
why DAAL_CHECK_MALLOC is not used here?

Reply:
It's lambda expression. It has to return TGroupRes pointer, not Status.